.github/workflows/presubmit.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           git remote add upstream https://github.com/taichi-dev/taichi.git
           git fetch upstream master
           sudo apt-get install clang-format
-          python3 -m pip install --user yapf gitpython colorama isort
+          python3 -m pip install --user -r requirements_dev.txt
```

Comment:
Will this make this step too slow? IMO checking code format needs to be as fast as possible.

Reply:
Hmmm good point, IIRC this isn't too slow but yea, let's just explicitly install it then. Just another call site to change manually :P